src/errors.rs

```diff
@@ -102,31 +102,6 @@ impl MomentoError {
     }
 }
 
-trait ErrorDetails {
-    fn message(&self) -> &String;
-    fn error_code(&self) -> &MomentoErrorCode;
-    fn inner_error(&self) -> Option<&ErrorSource>;
-    fn details(&self) -> Option<&MomentoGrpcErrorDetails>;
-}
-
-impl ErrorDetails for MomentoError {
-    fn message(&self) -> &String {
-        &self.message
-    }
-
-    fn error_code(&self) -> &MomentoErrorCode {
-        &self.error_code
-    }
-
-    fn inner_error(&self) -> Option<&ErrorSource> {
-        self.inner_error.as_ref()
-    }
-
-    fn details(&self) -> Option<&MomentoGrpcErrorDetails> {
-        self.details.as_ref()
-    }
-}
```

Comment:
@anitarua I didn't especially want to delete this stuff, but I was getting a compiler warning before, and then when I started trying to run `make` it was giving me a full-on error about this. not sure what this stuff is for or if it's really okay to delete it.

Reply:
I think I had basically made a duplicate called `MomentoGrpcErrorDetails` when I worked on the error stuff a while ago, I think it should be find to delete this

Reply:
okay. idk why this is failing for me but not for y'all :)